src/backend/distributed/utils/statistics_collection.c

```diff
@@ -112,46 +133,41 @@ CollectBasicUsageStatistics(void)
 	appendStringInfo(fields, ",\"node_metadata\": %s", metadataJsonbStr);
 	appendStringInfoString(fields, "}");
 
-	CommitTransactionCommand();
-
 	return SendHttpPostJsonRequest(STATS_COLLECTION_HOST "/v1/usage_reports",
 								   fields->data, HTTP_TIMEOUT_SECONDS);
 }
 
 
 /*
- * ClusterSize returns total size of data store in the cluster consisting of
- * given distributed tables. We ignore tables which we cannot get their size.
+ * DistributedTablesSize returns total size of data store in the cluster consisting
+ * of given distributed tables. We ignore tables which we cannot get their size.
  */
 static uint64
-ClusterSize(List *distributedTableList)
+DistributedTablesSize(List *distributedTableList)
 {
-	uint64 clusterSize = 0;
+	uint64 totalSize = 0;
 	ListCell *distTableCacheEntryCell = NULL;
 
 	foreach(distTableCacheEntryCell, distributedTableList)
 	{
 		DistTableCacheEntry *distTableCacheEntry = lfirst(distTableCacheEntryCell);
 		Oid relationId = distTableCacheEntry->relationId;
-		MemoryContext savedContext = CurrentMemoryContext;
 
-		PG_TRY();
+		/*
+		 * Ignore hash partitioned tables with size greater than 1, since
+		 * cstore_table_size() doesn't work on them.
```

Comment:
I guess `cstore_table_size` should be `citus_*` ?

Reply:
Done.